x/emissions/keeper/keeper.go

```diff
@@ -95,11 +94,11 @@ type Keeper struct {
 	// map of (reputer) -> removal information for that reputer
 	stakeRemoval collections.Map[collections.Pair[TopicId, ActorId], types.StakeRemoval]
 	// map of (delegator) -> removal information for that delegator
-	delegateStakeRemoval collections.Map[collections.Triple[TopicId, ActorId, Delegator], types.DelegateStakeRemoval]
+	delegateStakeRemoval collections.Map[collections.Triple[TopicId, ActorId, ActorId], types.DelegateStakeRemoval]
```

Comment:
dumb question, why are we using strings for addresses in the keeper at all?

can we just use the binary data type in our data structures? Is this because we don't have an appropriate encoding scheme for cosmos.collections? I imagine we'd get speed benefits not using strings, just store binary data of known size